task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -1135,6 +1135,7 @@ def test_get_variable_from_context(
             "push_task",
             ["push_task1", "push_task2"],
             {"push_task1", "push_task2"},
+            ("push_task1", "push_task2"),
```

Comment:
This doesn't really tear the end to end flow as it mocks most of the path I think 🤔